content/en/security/application_security/threats/setup/compatibility/php.md

```diff
@@ -97,20 +97,16 @@ The following frameworks aren't directly instrumented by Application Security, b
 | CakePHP       | 2.x       |  {{< X >}} | {{< X >}} |
 | CodeIgniter   | 2.x       |  {{< X >}} | {{< X >}} |
 | FuelPHP       | 1.1        |  {{< X >}} | {{< X >}} |
-| Laravel       | 4.2, 5.x, 6.x        | {{< X >}} | {{< X >}} |
-| Laravel 8     | 8.x (tracer 0.52.0+) | {{< X >}} | {{< X >}} |
+| Laravel       | 4.2, 5.x, 6.x, 8x(tracer 0.52.0+), 9x, 10x        | {{< X >}} | {{< X >}} |
 | Lumen         | 1.9-2.29    |  {{< X >}} | {{< X >}} |
 | Magento       |  3.8+       |  {{< X >}} | {{< X >}} |
 | Neos Flow     |  3.0.x      |  {{< X >}} | {{< X >}} |
 | Phalcon       | 3.1+        |  {{< X >}} | {{< X >}} |
 | Slim          | 3.1+        |  {{< X >}} | {{< X >}} |
-| Symfony 3     | 3.1+        |  {{< X >}} | {{< X >}} |
-| Symfony 4     | 3.1+        |  {{< X >}} | {{< X >}} |
-| Symfony 5     | 3.1+        |  {{< X >}} | {{< X >}} |
-| Wordpress     | 3.1+        |  {{< X >}} | {{< X >}} |
+| Symfony     | 3.1+, 4x, 5x, 6x        |  {{< X >}} | {{< X >}} |
+| Wordpress     | 3.1+, 4x, 5x, 6x    |  {{< X >}} | {{< X >}} |
```

Comment:
The nomenclature seems broken, in some cases you use a dot between the major and the `x` minor, but in some  you don't.